docs/platform/concepts/service_backups.rst

```diff
@@ -146,6 +146,33 @@ Aiven for InfluxDB®
 
 Aiven for InfluxDB backups are taken every 12 hours with 2.5 days of retention. InfluxDB® is automatically backed up, encrypted, and uploaded to the Aiven's S3 account in the same region. When an instance has to be rebuilt, the backup is downloaded and restored to create a new instance.
 
+Aiven for ClickHouse®
+'''''''''''''''''''''
+
+Aiven for ClickHouse® provides automatic daily backups. The `Astacus <https://github.com/aiven/astacus>`_ backup manager for distributed databases runs on all nodes to coordinate backups of cluster databases.
+
+Each file to be backed up is encrypted, compressed, and uploaded to an object storage (Amazon S3 or Google Cloud Storage).
+
+Aiven for ClickHouse backups cover
```

Comment:
Do we need the whole list here? Other services don't go into this much detail on this page.
I feel like it would be more adapted to the [Disaster recovery / Backup & Restore](https://docs.aiven.io/docs/products/clickhouse/concepts/disaster-recovery#backup-and-restore) section (and vice-versa, your wording there is concise and might be adapted here).
What do you think?

Reply:
@aris-aiven tried to follow your hint